ArrtSource/Rendering/UI/SceneState.cpp

```diff
@@ -378,6 +382,14 @@ void SceneState::UpdateProjectionMatrix()
     m_perspectiveMatrixInverse = m.inverted();
 }
 
+void SceneState::UpdatePointSize()
+{
+    if (m_client)
+    {
+        m_client->GetPointCloudSettingsExperimental()->SetPointSizeScale(m_arrOptions->GetPointSize() / 10.0f);
```

Comment:
This /10 conversion is also in multiple parts of the code.
Maybe add a m_arrOptions->GetPointSizeFloat() option and fix this factor of 10 there at one point?